airflow/executors/mesos_executor.py

```diff
@@ -0,0 +1,422 @@
+    """
+
+    # pylint: disable=super-init-not-called
+    def __init__(self, executor, task_queue, result_queue, task_cpu: int = 1, task_mem: int = 256):
+        self.task_queue = task_queue
+        self.result_queue = result_queue
+        self.task_cpu = task_cpu
+        self.task_mem = task_mem
+        self.task_counter = 0
+        self.task_key_map: Dict[str, str] = {}
+        self.log = executor.log
+        self.client = executor.client
+        self.executor = executor
+        self.driver = None
+
+        if not conf.get('mesos', 'DOCKER_IMAGE_SLAVE'):
+            self.log.error("Expecting docker image for  mesos executor")
+            raise AirflowException("mesos.slave_docker_image not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_VOLUME_DRIVER'):
+            self.log.error("Expecting docker volume driver for mesos executor")
+            raise AirflowException("mesos.docker_volume_driver not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_VOLUME_DAG_NAME'):
+            self.log.error("Expecting docker volume dag name for mesos executor")
+            raise AirflowException("mesos.docker_volume_dag_name not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_VOLUME_DAG_CONTAINER_PATH'):
+            self.log.error("Expecting docker volume dag container path for mesos executor")
+            raise AirflowException("mesos.docker_volume_dag_container_path not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_VOLUME_LOGS_NAME'):
+            self.log.error("Expecting docker volume logs name for mesos executor")
+            raise AirflowException("mesos.docker_volume_logs_name not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_VOLUME_LOGS_CONTAINER_PATH'):
+            self.log.error("Expecting docker volume logs container path for mesos executor")
+            raise AirflowException("mesos.docker_volume_logs_container_path not provided for mesos executor")
+
+        if not conf.get('mesos', 'DOCKER_SOCK'):
+            self.log.error("Expecting docker sock path for mesos executor")
+            raise AirflowException("mesos.docker_sock not provided for mesos executor")
+
+        self.mesos_slave_docker_image = conf.get('mesos', 'DOCKER_IMAGE_SLAVE').replace('"', '')
+        self.mesos_docker_volume_driver = conf.get('mesos', 'DOCKER_VOLUME_DRIVER').replace('"', '')
+        self.mesos_docker_volume_dag_name = conf.get('mesos', 'DOCKER_VOLUME_DAG_NAME').replace('"', '')
+        self.mesos_docker_volume_dag_container_path = conf.get(
+            'mesos', 'DOCKER_VOLUME_DAG_CONTAINER_PATH'
+        ).replace('"', '')
+        self.mesos_docker_volume_logs_name = conf.get('mesos', 'DOCKER_VOLUME_LOGS_NAME').replace('"', '')
+        self.mesos_docker_volume_logs_container_path = conf.get(
+            'mesos', 'DOCKER_VOLUME_LOGS_CONTAINER_PATH'
+        ).replace('"', '')
```

Comment:
What are all the `replace()` here for?

Reply:
I want to be sure, that there will be no " character in the string. If conf.get already cleanup or escape, then I'm fine and remove it.

Reply:
Well, I have to cleanup the values. If I don't do that, mesos can not schedule the task because of unsupported characters.